docs/data-sources/dnsaas_record.md

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "DNS as a Service"
+layout: "ionoscloud"
+page_title: "IonosCloud : ionoscloud_dns_record"
+sidebar_current: "docs-dns_record"
+description: |-
+  Get information on a DNS Record.
+---
+
+# ionoscloud_dns_zone
```

Comment:
ionoscloud_dns_record*

Reply:
@mimihalescu done, thanks for catching this!